test/new-e2e/runner/local-profile.go

```diff
@@ -36,7 +36,7 @@ func NewLocalProfile() (Profile, error) {
 	} else {
 		store = parameters.NewCascadingStore(envValueStore)
 	}
-	return localProfile{baseProfile: newProfile("e2elocal", []string{"aws/sandbox"}, store, nil)}, nil
+	return localProfile{baseProfile: newProfile("e2elocal", []string{"aws/sandbox"}, store, &store)}, nil
```

Comment:
Is there a reason for this change, as `newProfile` uses `store` when `secretStore` is nil?

Reply:
Some parameters are currently looked up only in the secret store, like the api key

Reply:
@ogaca-dd is right, it's not necessary, fallback happens automatically.

Reply:
Saw it now, handled by profile inner constructor, changing this